### PR TITLE
[ZH]: translation

### PR DESCRIPTION
**Pull request by lovecity08**
Chinese version of White Paper

### CODE REVIEW

**Comment by mfts** [comment]
Hi @lovecity08 

I noticed that you are replacing the original `white-paper.md` with your translated version. Please **don't delete** the original `white-paper.md` file, instead please create a folder in the root repository called `i18n` and within that another folder with your language ISO code `zh` (see below):
```
...
i18n/
  zh
...
```
Then please place both the translated files `white-paper.md` and `metadata.yml` in the newly created folder.

**Comment by mfts** [comment]
Additionally, I saw that you opened two PRs; one for each file. Please add both `metadata.yml` and `white-paper.md` in the same PR. 

The second PR is here: #79

**Review by mfts — CHANGES_REQUESTED** [review]
Please fix the small details in the `metadata.yml` file

**Comment by mfts** [comment]
@Xercesblu3 please run workflow

File: i18n/zh/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+title: 用于奖励开源生态系统的去中心化协议
+abstract: |-
+ 为所有开源软件创建一个开放、公共和稳定的注册表将使项目能够独立发布版本，而不是
+ 依赖于将这些不规则数据组装到数百个独立（和重复）系统中的第三方。Package维护者将
+ 他们的作品发布到由拜占庭容错区块链驱动的去中心化注册表，以消除单一故障源，提供
+ 稳定的发布，并允许社区来管理其开源生态系统的区域，而不受外部议程的影响。
```
Comment (mfts) [inline i18n/zh/metadata.yml:6]:
```suggestion
  为所有开源软件创建一个开放、公共和稳定的注册表将使项目能够独立发布版本，而不是
  依赖于将这些不规则数据组装到数百个独立（和重复）系统中的第三方。Package维护者将
  他们的作品发布到由拜占庭容错区块链驱动的去中心化注册表，以消除单一故障源，提供
  稳定的发布，并允许社区来管理其开源生态系统的区域，而不受外部议程的影响。
```

Please properly indent this

File: i18n/zh/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- id: npmjsLogFourjs
+  url: https://www.npmjs.com/package/log4js/
+- id: arxiv
+  url: https://arxiv.org/abs/1207.2617/
+- id: web3
+  url: https://research.web3.foundation/en/latest/polkadot/overview/2-token-economics.html
+lang: zh
+dir: ltr 
+header-includes:
+- \fancyfoot[L]{${VERSION 1}+${zh}}
```
Comment (mfts) [inline i18n/zh/metadata.yml:65]:
```suggestion
- \usepackage{fancyhdr,ragged2e}
- \lhead{\parbox[t]{0.5\textwidth}{\RaggedRight\rightmark\strut}}
- \rhead{\parbox[t]{0.5\textwidth}{\RaggedLeft\leftmark\strut}}
- \setlength{\headheight}{5\baselineskip}
- \pagestyle{fancy}
- \fancyfoot[LE,RO]{© 2022 tea.inc.}
- \fancyfoot[L]{v1.0.3+zh}
```

add missing header-includes items

**Review by mfts — CHANGES_REQUESTED** [review]
Small fixes in the metadata.yml file

**Comment by mfts** [comment]
@jhheider Georgia doesn't support Chinese characters #90

**Comment by Xercesblu3** [comment]
@mfts it's not letting me run. Possibly because of the failed checks?

**Comment by jhheider** [comment]
> @jhheider Georgia doesn't support Chinese characters #90 

Ugh. Hopefully we can find a single font that does what we need.

**Comment by mfts** [comment]
> @mfts it's not letting me run. Possibly because of the failed checks? 

I think that's because I'm requesting changes from the PR author.

**Comment by mfts** [comment]
Thank you @lovecity08 for your contribution. Everything works well. I merged your contribution into a new PR to add the font for Chinese.

Merged into #92

**Pull request closed without merging**